src/main/java/io/reactivex/Flowable.java

```diff
@@ -10650,7 +10650,7 @@ public final Completable ignoreElements() {
     @CheckReturnValue
     @BackpressureSupport(BackpressureKind.UNBOUNDED_IN)
     @SchedulerSupport(SchedulerSupport.NONE)
-    public final Maybe<T> reduce(BiFunction<T, T, T> reducer) {
+    public final Maybe<T> reduce(BiFunction<? super T, ? super  T, ? extends T> reducer) {
```

Comment:
`reduce(T, T, T)` was discussed quite a while back and it has to stay non-variant. One of the reasons is the limits of javac and what @davidmoten showed.